### ISSUE TITLE
Tunes on image block are discarded

### ISSUE THREAD

**Issue by brtchkv** [2ik/django-editorjs-fields#29]
Faced very weird bug, when I add tune blockAlignment to the Image tool, it gets displayed only once shortly after the redeploy. Then it is gone until next redeploy is done. I tried to clean cache, open window in different browsers, in private modes -- nothing helps. 

It is visible only once (and present in html) and then it's gone till the platform is restarted. Seems to be some issue with adding Tunes blocks to Image in particular, since on any other block it works well.

In browser console logs there's no errors. And in backend logs -- too.

```
  plugins = [ '@editorjs/image', 'editorjs-text-alignment-blocktune' ],
  tools = {
            'Image': {
                  'class': 'ImageTool',
                  'tunes': ['blockAlignment'],
                  'inlineToolbar': True,
                  "config": {
                      "endpoints": {
                          "byFile": reverse_lazy('editorjs_image_upload'),
                          "byUrl": reverse_lazy('editorjs_image_by_url')
                      }
                  },
              },
            'blockAlignment': {
                  'class': 'AlignmentBlockTune',
                  'config': {
                      'default': "left",
                      'blocks': {
                          'header': 'left',
                          'list': 'left',
                          'Image': 'center'
                      }
                  },
              }
         }           
```
First view:

<img width="785" alt="image" src="https://user-images.githubusercontent.com/17526135/147418283-b952c944-4397-4bc5-9841-f4ff0ab6a2f2.png">

Any other view in any condition (user switch, cache clean, different browser):

<img width="732" alt="image" src="https://user-images.githubusercontent.com/17526135/147418305-6e287f5b-4035-4a56-b49f-f4fae90d3765.png">

Same alignment block on any other block for example Paragraph:

<img width="736" alt="image" src="https://user-images.githubusercontent.com/17526135/147418356-ee781e8d-8df6-406e-af38-f2a6fa0095d9.png">

**Comment by brtchkv** [comment]
Hi, @2ik, can you give estimation if this will be fixed soon (till mid-January). Or, I can fork and fix it on my own or maybe try another image block component with hard coded tunes?

**Comment by 2ik** [comment]
Hi, I can't find the time to work on it yet
Has testing been done on pure html + js (without Django) ?
Where can be errors in django-editorjs-fields?

I'm always happy new PR ;)

**Comment by brtchkv** [comment]
Hi, @2ik, I ended up forking editor js image component and hard-coding in it the tunes I needed :)

About pure html and js, I tried. but got somewhere stuck and decided that just forking original component and rewrite it a bit for myself would be faster and less painful )0))

And s nastupayushhim novy`m godom! 🎅 ❄️
